src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BOUND = 101;
+    public static final Random random = new Random();
+    public static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    public final Random random = new Random();
    public final ColorSupplier colorSupplier = new ColorSupplier();
```
these dependency injections shouldn't be static

Reply:
Yes, and this is the next lecture and at the time of completing this task I don’t know this